CLAUDE.md

```
@@ -0,0 +1,112 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Project Overview
+
+Kinobi is a code generator for Solana programs that generates type-safe client code from IDL (Interface Definition Language) files. It parses Anchor and Shank IDLs and can generate JavaScript (Umi-based), JavaScript (web3.js 2.0-based), and Rust clients.
+
+## Development Commands
+
+### Build
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)